src/api/questions.js

```diff
@@ -298,41 +298,52 @@ router.delete(
 // Mark the question as answered
 router.post(
   '/:questionId/answered',
-  [
-    requireCourseStaffForQueueForQuestion,
-    requireQuestion,
-    check('preparedness').isIn(['bad', 'average', 'good']),
-    check('comments')
-      .optional({ nullable: true })
-      .trim(),
-    failIfErrors,
-  ],
+  [requireCourseStaffForQueueForQuestion, requireQuestion, failIfErrors],
   safeAsync(async (req, res, _next) => {
-    const data = matchedData(req)
-
-    // Temporary, easy fix to avoid having to rename enums
-    // TODO Fix this garbage
-    let mappedPreparedness = data.preparedness
-    switch (data.preparedness) {
-      case 'bad':
-        mappedPreparedness = 'not'
-        break
-      case 'good':
-        mappedPreparedness = 'well'
-        break
-      default:
-        break
-    }
+    const { shouldCheckFeedback } = req.body
```

Comment:
Whether or not feedback is checked shouldn't be provided by the client - this should be enforced (or not) based on what the queue is configured as in the DB.